src/mame/drivers/nes_vt.cpp

```diff
@@ -75,6 +76,7 @@ class nes_vt_base_state : public driver_device
 	uint8_t upper_412c_r();
 	uint8_t upper_412d_r();
 	void upper_412c_w(uint8_t data);
+	int m_previous_port0;
```

Comment:
Should this be changed to `uint8_t` and registered for save states?  (I’m not sure whether saving it would be useful, considering only one of the state classes in this file saves any members at all.)

Reply:
yeah, adding states / verifying them etc. is on my todo list.  at the moment I'm just trying to identify broken behaviours and address them.